app/cdap/components/ConnectionList/Components/ConnectionTabs/types.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Dispatch, SetStateAction } from 'react';
+
+export interface IRenderLabelProps {
+  index: number;
+  connectorType: IConnectorTabType;
+  connectionIdProp: string;
+  toggleLoader: (value: boolean, isError?: boolean) => void;
+  setIsErrorOnNoWorkSpace: Dispatch<SetStateAction<boolean>>;
+}
+
+export interface IConnectorTabType {
```

Comment:
somewhere it is connectiontab
here it is connectortab

plz maintain clear consistency so we know what we are referring to
let me know if u need more info on what is connector type and what is connection

:)

Reply:
Done